### PR TITLE
SEO-176927-Vue-Page-With-Redirect-Links

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
Please find and review the changes made.
Regards,
Scholar.

### CODE REVIEW

File: ej2-vue/diagram/interaction.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ Clicking and dragging the diagram area allows to create a rectangular region. Th
 
 The client-side methods [`select`](https://ej2.syncfusion.com/vue/documentation/api/diagram/#select) and [`clearSelection`](https://ej2.syncfusion.com/vue/documentation/api/diagram/#clearselection) help to select or clear the selection of the elements at runtime. The following code example illustrates how to select or clear the selection of an item using program.
 
-Get the current selected items from the [`nodes`](https://ej2.syncfusion.com/vue/documentation/api/diagram/selectorModel#nodes-nodemodel[]) and [`connectors`](https://ej2.syncfusion.com/vue/documentation/api/diagram/selectorModel#connectors-connectormodel[]) collection of the [`selectedItems`](https://ej2.syncfusion.com/vue/documentation/api/diagram/#selecteditems) property of the diagram model.
+Get the current selected items from the [`nodes`](https://ej2.syncfusion.com/vue/documentation/api/diagram/selectorModel/#nodes-nodemodel%5B%5D) and [`connectors`](https://ej2.syncfusion.com/vue/documentation/api/diagram/selectorModel/#connectors-connectormodel%5B%5D) collection of the [`selectedItems`](https://ej2.syncfusion.com/vue/documentation/api/diagram/#selecteditems) property of the diagram model.
```
Comment (Yvone-Atieno) [inline ej2-vue/diagram/interaction.md:48]: Revert changes

File: ej2-vue/diagram/interaction.md [hunk 2]
```diff
@@ -137,28 +137,28 @@ Diagram provides support to drop a node/connector over another node/connector. T
 
 ## User handles
 
-* User handles are used to add some frequently used commands around the selector. To create user handles, define and add them to the [`userHandles`](https://ej2.syncfusion.com/vue/documentation/api/diagram/selectorModel#userHandles-userhandlemodel[]) collection of the [`selectedItems`](https://ej2.syncfusion.com/vue/documentation/api/diagram/#selecteditems) property.
+* User handles are used to add some frequently used commands around the selector. To create user handles, define and add them to the [`userHandles`](https://ej2.syncfusion.com/vue/documentation/api/diagram/selectorModel/#userHandles-userhandlemodel%5B%5D) collection of the [`selectedItems`](https://ej2.syncfusion.com/vue/documentation/api/diagram/#selecteditems) property.
```
Comment (Yvone-Atieno) [inline ej2-vue/diagram/interaction.md:140]: Revert changes and check in all places

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-vue/diagram/interaction.md (hunk 2 above)
Comment (Scholar-jaruha) [reply ej2-vue/diagram/interaction.md]: reverted

File: ej2-vue/diagram/interaction.md (hunk 1 above)
Comment (Scholar-jaruha) [reply ej2-vue/diagram/interaction.md]: reverted

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**